app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkController.kt

```diff
@@ -169,7 +169,10 @@ class DefaultBookmarkController(
     override fun handleRequestSync() {
         scope.launch {
             store.dispatch(BookmarkFragmentAction.StartSync)
-            activity.components.backgroundServices.accountManager.syncNow(SyncReason.User)
+            activity.components.backgroundServices.accountManager.syncNow(
```

Comment:
I'm wondering if there will be problems when requests are debounced that have different subsets of engine requests. For example this request is a full sync, can it get overwritten by a tabs-only sync from elsewhere?

Reply:
I'll look into this.  Thanks for pointing it out.